site/topics/values-variables-types/values-variables-types.rst

```diff
@@ -93,71 +93,64 @@ Terminology
 * Compiling
 * Interpreter vs IDE
    
-   
-Okay, we're done with the background, let's get on with the real stuff
-======================================================================
+
+Print
+=====
 
 .. admonition:: Activity
 
-    Write a (single-line) Python program that prints a witty message, of your choice.   
-   
+    Write a (single-line) Python program that prints a witty message of your choice.
 
-print
-=====
 
 * Print is a *function* that allows us to print out information to the screen
 * Print might end up being your best friend
 * Get used to writing it
-   
-   
-Values 
-======
+
+
+Values And Types
+================
 
 * Values are things that a program manipulates
-    * *Strings*: "abcdef"
-    * *Integers*: 7, 42, 97
-    * *Floating-point numbers*: 3.792, 0.000000000005
+    * *Strings*: ``"abcdefg"``, ``"Hello World"``
+    * *Integers*: ``7``, ``42``, ``97``
+    * *Floating-point numbers*: ``3.792``, ``0.000000000005``
+
 * These values are called **literals**
-    * like, 1 is *literally* 1  
+    * Like, ``1`` is *literally* ``1``
+
 * Notice how I described the **type** of each value along with the value itself
     * Strings
     * Integers
     * Float
    
-* Computers are exceptionally stupid. You must be completely explicit about everything
+* To a computer, the integer ``1`` is not necessarily the same thing as the floating point number ``1.0`` or the string ``"1"``
 
-* To a computer, the integer 1 is not necessarily the same thing as the floating point number 1.0... because they have different *types*
-    * They actually have different meaning
-    * They even technically have different physical representations inside the computer too, which is neato  
-* Many of the errors you will make in programming result from mixing types inappropriately
-* Some languages (e.g., C, Fortran, Java) are very militant about types. You have to be totally explicit about them
-* Python is a little more relaxed. You *can* be explicit, but you don't have to be. Python will guess if you don't tell it
-* Upside: less to worry about and less clutter in your code
+* Some of the errors you will make will be a result from mixing types incorrectly
+* Some languages (e.g., C, Java) are strict about types
+    * You have to be totally explicit about them
 
+* Python is a little more relaxed
+    * Python will guess what the type is
```

Comment:
It's not really a guess. All data is always typed (sort of). That type may not be known at compile time is all.